OSPSuite Architecture.md

```diff
@@ -0,0 +1,280 @@
+# OSPSuite Architecture
+
+## Introduction
+
+In this part we will describe the Open Systems Pharmacology Suite, a set of powerful and easy-to-use modeling & simulation tools for pharmaceutical and other life-sciences applications. Qualified and accepted by the scientific community including academia, regulatory agencies and industry. This includes the main parts of the Suite and their architecture. Namely we have the [OSPSuite Core](https://github.com/Open-Systems-Pharmacology/OSPSuite.Core) where the main functionalities of the Suite reside that are common in all other applications, [PK-Sim](https://github.com/Open-Systems-Pharmacology/PK-Sim),  a comprehensive software tool for whole-body physiologically based pharmacokinetic modeling and [MoBi](https://github.com/Open-Systems-Pharmacology/MoBi), a software tool for multiscale physiological modeling and simulation that offers  more freedom for the user to import and set up models from scratch. Additionally, [OSPSuite-R](https://github.com/Open-Systems-Pharmacology/OSPSuite-R) is a package that package provides the functionality of loading, manipulating, and simulating the simulations created in the Open Systems Pharmacology Software tools PK-Sim and MoBi in the R programming language. There are additional parts to the OSPSuite universe, f.e. [OSPSuite.SimModel](https://github.com/Open-Systems-Pharmacology/OSPSuite.SimModel) that reads the model description in its XML format, creates a differential equations system from it and solves it. 
+
+The structure of the OSPSuite Architecture can be visualised as follows:
+
+![The OSPSuite Suite structure](assets/images/ospsuite_architecture_diagram.png)
+
+
+
+## Technology used
+
+The Core of the OSPSuite as well as PKSim and MoBi are written using the .NET Framework currently in version 4.8. The language used is C# 7.3.
+The architecture is based on the “onion architecture” and the view pattern used is the MVP pattern (MVC, MVVM etc.)
```

Comment:
```The architecture is based on the “onion architecture” and the view pattern used is the MVP pattern (MVC, MVVM etc.)```

What about

```
The architecture is based on the “onion architecture” and the view pattern used is the MVP pattern (Model-View-Presenter)
```
I would level out the other types of Model-View designs

Reply:
done